src/vma/sock/fd_collection.cpp

```diff
@@ -90,6 +90,12 @@ fd_collection::~fd_collection()
 {
 	fdcoll_logfunc("");
 
+	for (int i = 0; i < m_n_fd_map_size; i++) {
+		if (m_p_cq_channel_map[i]) {
+			delete m_p_cq_channel_map[i];
+		}
+	}
+
```

Comment:
@serglelv look inside clear() call 2 lines below. It does your code in lines 93-98
So apparantly your code is redundant. However clear() is checking m_n_fd_map_size having a positive value and just before calling clean() this variable is set to -1. 
```
m_n_fd_map_size = -1;
```
So it looks like a bug that needs to be fixed.
Please update